servers/rendering/rasterizer.h

```diff
@@ -88,6 +88,13 @@ class RasterizerScene {
 	virtual void environment_glow_set_use_bicubic_upscale(bool p_enable) = 0;
 	virtual void environment_set_fog(RID p_env, bool p_enable, float p_begin, float p_end, RID p_gradient_texture) = 0;
 
+	virtual void environment_set_volumetric_fog(RID p_env, bool p_enable, float p_density, const Color &p_light, float p_light_energy, float p_lenght, float p_detail_spread, float p_gi_inject, RS::EnvVolumetricFogShadowFilter p_shadow_filter) = 0;
+
+	virtual void environment_set_volumetric_fog_volume_size(int, int) = 0;
```

Comment:
I guess add names to the parameters

Reply:
Small typo:
p_lenght instead of p_length

Reply:
Should be p_length instead of p_lenght